spec/server/functional/level_session.spec.coffee

```diff
@@ -64,3 +65,99 @@ describe '/db/level.session', ->
       expect(err).toBeNull()
       expect(res.statusCode).toBe 404
       done()
+
+      
+describe 'POST /db/level.session/:handle/submit-to-ladder AND POST /queue/scoring', ->
```

Comment:
Yay Tests!  But it looks like we don't actually reach into the DB and see if anything is actually queued.  Is this really tricky to do with level sessions?